src/main/kotlin/com/approvers/devlazaApi/controller/ExtendsFunctions.kt

```diff
@@ -0,0 +1,85 @@
+package com.approvers.devlazaApi.controller
+
+import com.approvers.devlazaApi.errors.BadRequest
+import com.approvers.devlazaApi.errors.NotFound
+import com.approvers.devlazaApi.model.ProjectMember
+import com.approvers.devlazaApi.model.Tags
+import com.approvers.devlazaApi.model.TagsToProjectsBridge
+import com.approvers.devlazaApi.model.Token
+import com.approvers.devlazaApi.repository.ProjectMemberRepository
+import com.approvers.devlazaApi.repository.TagsRepository
+import com.approvers.devlazaApi.repository.TagsToProjectsBridgeRepository
+import com.approvers.devlazaApi.repository.TokenRepository
+import java.util.*
+
+fun String.convertToUUID():UUID{
```

Comment:
これは慣習ですがconvertはあまりこの場面で使うべき単語ではないので普通に`toUuid`もしくは`toUUID`の方がいい。
```suggestion
fun String.toUUID():UUID{
```